kolibri/content/test/test_content_app.py

```diff
@@ -193,6 +193,23 @@ def test_channelmetadata_resource_info(self):
         self.assertEqual(response.data['on_device_resources'], 3)
         self.assertEqual(response.data['on_device_file_size'], 0)
 
+    def test_channelmetada_langfield(self):
```

Comment:
is this a typo: `channelmetada` (`channelmetadata`)?

Reply:
ohh yes! It's a typo. Thank you for pointing it out! I'll fix it soon